backend/routes/logout.ts

```diff
@@ -13,7 +13,7 @@ logoutRouter.post('/logout', function (req, res, next) {
         next(err);
         return;
       }
-      res.redirect('http://localhost:3000/login');
+      res.redirect('http://localhost:3000/tempLandingPage');
```

Comment:
we don't have a templandingPage anymore. this can just redirect to '/'

Reply:
Yeah I believe you already changed this so it gets redirected to the login page. I can leave it like that because that also makes sense to do.